api/views.py

```diff
@@ -74,14 +75,17 @@ def added(request):
 def add_site(request):
     """Handle incoming POST requests to add new sites to the list.
 
-    For sites submitted via the extension.
+    Creates a new SiteRequest object for sites submitted via the extension.
 
-    Returns JSON response.
+    Returns JSON response back to the extension.
 
     """
 
     add_site = request.POST['site']
 
+    site_request = SiteRequest(website=add_site)
```

Comment:
So. Considering 
```
website = models.URLField(blank=True, null=True)
```
We may want to wrap this in a try/except? I'll test, but it might throw an error if the `add_site` is not a valid url.